wpeverest/everest-forms
# Fix - min/max value glitch and default value validation.

### All Submissions:






### Changes proposed in this Pull Request:
Mininum/Maximum value glitches resolved and Default value validation is  added for number and range slider field.




### How to test the changes in this Pull Request:

1. Create a form with at least one number field and one range slider field.
2. Go to Field Options of respective field.
3. Verify min/max value glitch is resolved or not. And, also verify default value validation.

### Types of changes:




### Other information:




### Changelog entry

> Fix - min/max value glitch and default value validation.
